### PR TITLE
Projects/website

### PR DESCRIPTION
**Pull request by EllisWaterman**

### CODE REVIEW

File: projects/website/public/distanceCalculator.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const calculate = (x1,x2,y1,y2) => {
+   return Math.sqrt(((x2-x1)**2)+((y2-y1)**2))
+}
+const roundToDecimalPlace = (number, places) => {
+   return Math.round(number*10**places)/10**places
```
Comment (gigamonkey) [inline projects/website/public/distanceCalculator.js:5]: Nice job figuring out a way to do this but there's a method on numbers to do this: `number.toFixed(places)` which returns a string with the number to the given number of places. See 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

for more details.

File: projects/website/public/distanceCalculator.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const calculate = (x1,x2,y1,y2) => {
+   return Math.sqrt(((x2-x1)**2)+((y2-y1)**2))
```
Comment (gigamonkey) [inline projects/website/public/distanceCalculator.js:2]: Although I've made you write this a billion times on various assignments, there's actually a method on `Math` that does this: `Math.hypot(x2 - x1, y2 - y1)`. That has the advantage of being more concise and also they way it is implemented deals better with very small and large values where the details of the way real numbers are approximating in computers can make calculations tricky.

**Review by gigamonkey — APPROVED** [review]
Nice. Just a few comments about built in functions and methods you can use.

File: projects/website/public/distanceCalculator.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+   return Math.round(number*10**places)/10**places
+    
+}
+const submit = document.querySelector('.submit')
+submit.onclick = () => {
+   let x1 = document.querySelector('#x1').value
+   let x2 = document.querySelector('#x2').value
+   let y1 = document.querySelector('#y1').value
+   let y2 = document.querySelector('#y2').value
+   document.querySelector('#output').innerHTML = roundToDecimalPlace(calculate(x1,x2,y1,y2),2)
```
[the remark this answers is not in the document]
Comment (EllisWaterman) [reply projects/website/public/distanceCalculator.js:14]: It did work before, but this also works and is probably easier to read.

**Pull request merged**